airflow/models.py

```diff
@@ -3024,6 +3024,24 @@ def get_active_runs(self, session=None):
         return active_dates
 
     @provide_session
+    def count_active_runs(self, external_trigger=None, session=None):
```

Comment:
get_num_active_runs is more standard also would be good to replace the other places that used the old query with this new function too:
/Users/dan_davydov/ws/airflow/airflow/ti_deps/deps/dagrun_exists_dep.py